### DISCUSSION TITLE
Getting a "Something went wrong" message for all the CodeSandbox examples

### DISCUSSION THREAD

**Discussion by alfonsoluis** [pmndrs/react-spring#1506] [Support] [answered]
Hi All, 

maybe this isn't related directly to the react spring codebase. But looking at the examples in CodeSandbox is my main resource of learning about the library since the documentation examples does not provide the full code. 

My main concern is not about this being a glitch with Code Sandbox, but something permanent that went broken. 

Not sure either if this is the place I should be looking for help on this issue. Hope is just a Glitch.

Thank's all

**Comment by joshuaellis** [comment, marked as answer]
The documentation does have full examples now? And editable ones too. 

Do you have an example of this error showing?

**Reply by alfonsoluis** [reply]
Seems to be working now. 

Each of the examples at the end of each documentation section have a link to CodeSandbox. 

For instance, in the case of the useTransition hook, there is a **List reordering** example with the link to [Try it on CodeSandbox](https://codesandbox.io/s/github/pmndrs/react-spring/tree/master/demo/src/sandboxes/list-reordering), pretty useful to understand a real use case. 

Anyway, I'm happy to report that the issue is no longer there. Thank you for looking into it :)

**Reply by joshuaellis** [reply]
I'm glad they're working 😅 I was aware of where these examples were, i rebuilt the docs for v9

**Reply by alfonsoluis** [reply]
I was lucky then, or is it just that this is an awesome project with a great community ;)
